lib/brains/management/commands/braintree_webhook.py

```diff
@@ -92,8 +125,11 @@ def webhook(self, url, kind, sub):
             # Braintree doesn't assume months are 30 days long.
             'paid': datetime.today() + timedelta(days=29),
             'next': datetime.today() + timedelta(days=30),
+            'transaction': transaction[kind],
+            'processor-response': processor_response[kind]
```

Comment:
I changed this and corresponding keys to `processor_response` in a later pull request.